examples/google-cloud-speech/output/google-cloud-speech/lib/google/cloud/speech/v1/speech_client.rb

```diff
@@ -0,0 +1,411 @@
+              config: config,
+              audio: audio
+            }.delete_if { |_, v| v.nil? }
+            req = Google::Gax::to_proto(req, Google::Cloud::Speech::V1::RecognizeRequest)
+            @recognize.call(req, options, &block)
+          end
+
+          # Performs asynchronous speech recognition: receive results via the
+          # google.longrunning.Operations interface. Returns either an
+          # `Operation.error` or an `Operation.response` which contains
+          # a `LongRunningRecognizeResponse` message.
+          #
+          # @param config [Google::Cloud::Speech::V1::RecognitionConfig | Hash]
+          #   *Required* Provides information to the recognizer that specifies how to
+          #   process the request.
+          #   A hash of the same form as `Google::Cloud::Speech::V1::RecognitionConfig`
+          #   can also be provided.
+          # @param audio [Google::Cloud::Speech::V1::RecognitionAudio | Hash]
+          #   *Required* The audio data to be recognized.
+          #   A hash of the same form as `Google::Cloud::Speech::V1::RecognitionAudio`
+          #   can also be provided.
+          # @param options [Google::Gax::CallOptions]
+          #   Overrides the default settings for this call, e.g, timeout,
+          #   retries, etc.
+          # @return [Google::Gax::Operation]
+          # @raise [Google::Gax::GaxError] if the RPC is aborted.
+          # @example
+          #   require "google/cloud/speech"
+          #
+          #   speech_client = Google::Cloud::Speech.new(version: :v1)
+          #   encoding = :FLAC
+          #   sample_rate_hertz = 44100
+          #   language_code = "en-US"
+          #   config = {
+          #     encoding: encoding,
+          #     sample_rate_hertz: sample_rate_hertz,
+          #     language_code: language_code
+          #   }
+          #   uri = "gs://bucket_name/file_name.flac"
+          #   audio = { uri: uri }
+          #
+          #   # Register a callback during the method call.
+          #   operation = speech_client.long_running_recognize(config, audio) do |op|
+          #     raise op.results.message if op.error?
+          #     op_results = op.results
+          #     # Process the results.
+          #
+          #     metadata = op.metadata
+          #     # Process the metadata.
+          #   end
+          #
+          #   # Or use the return value to register a callback.
+          #   operation.on_done do |op|
+          #     raise op.results.message if op.error?
+          #     op_results = op.results
+          #     # Process the results.
+          #
+          #     metadata = op.metadata
+          #     # Process the metadata.
+          #   end
+          #
+          #   # Manually reload the operation.
+          #   operation.reload!
+          #
+          #   # Or block until the operation completes, triggering callbacks on
+          #   # completion.
+          #   operation.wait_until_done!
+
```

Comment:
Why do some of these have a blank line while others do not?

Reply:
Good catch, I'll standardize the docs formatting.

Reply:
Fixed.